### PR TITLE
City states button influence bar

### PR DESCRIPTION
**Pull request by r3versi**
influence_bar

I also changed the thresholds of influence for city states, because the state `Unforgivable` wasn't "reachable" in practice:
* `< 30 || atWar` -> `Unforgivable`
* `<0` -> `Enemy`
* `< 30` -> `Neutral` 
* `< 60` -> `Friend`
* `>60 && allied` -> `Ally`

### CODE REVIEW

File: core/src/com/unciv/ui/utils/ImageGetter.kt [hunk 1]
```diff
@@ -256,6 +258,63 @@ object ImageGetter {
         return advancementGroup
     }
 
+    fun getInfluenceBar(influence: Float, relationshipLevel: RelationshipLevel, influenceBarSize: Float = 100f): Table {
```
Comment (yairm210) [inline core/src/com/unciv/ui/utils/ImageGetter.kt:261]: This is *only* used by the city button, so it should be a function in the city button

**Comment by r3versi** [comment]
I was thinking of adding it to the diplomacy screen (with slightly
different width and height). What do you think?

**Comment by yairm210** [comment]
I still feel that the ImageGetter is getting very clogged.
If that's the case, then I would still advise that the function goes in the CityButton, but in a companion object

**Comment by r3versi** [comment]
I see, but consider that ImageGetter is slim wrt to other classes. Anyway
I'm moving it to the CityButton although that's not the "semantically"
right place imho.

**Comment by r3versi** [comment]
OK, moved the method to CityButton and added it to DiplomacyScreen. Here is a preview
Screenshot from 2020-10-09 15-50-54

**Pull request merged**